evil-owl.el

```diff
@@ -96,7 +96,8 @@ Possible format specifiers are:
 - %m: the mark
 - %l: the mark's line number
 - %c: the mark's column number
-- %b: the mark's buffer"
+- %b: the mark's buffer
+- %s: text surrounding the mark"
```

Comment:
If you want, you can set the default formats to what you put in the readme. Totally up to you though.